### ISSUE TITLE
Wrong cursor working logic on characters delete

### ISSUE THREAD

**Issue by pavlo-vasylkivskyi-scx** [bl00mber/react-phone-input-2#273]
Hi, I found a little bug with input cursor
**Steps to reproduce:**

1. Fill in the phone number in the Phone number field (e.g.  12345678910)
2. Clear first 2 numbers
3. Press backspace button one more time

**After step 3, the cursor moves to the end of the line**
I searched in your docs for some solution, but can't find anything

**Comment by bill-oneil** [comment]
I have this bug too.

**Comment by yanosh-igor** [comment]
@bl00mber Are you planning to check this issue?

**Comment by bill-oneil** [comment]
I tried to press backspace to delete characters step by step, but the first letter of phone txt was not delete.


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by yanosh-igor** [comment]
@bl00mber 
Hey, I'm expecting keeping cursor at the same position where characters were deleted.
It moves to the end of the line right now.

**Comment by ramkychowdary** [comment]
Yes we I am also facing same issue. Can you please provide me input how to fix this issue.

**Comment by bill-oneil** [comment]
Excuse me, I didn’t fix this issue too.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
